packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -504,14 +508,20 @@ Media.stopStream = (stream) => {
    * 2) Firefox requires the stream to be stopped
    */
   return Promise.resolve().then(() => {
+    let didStopMedia = false;
+
     if (stream.getTracks) {
       stream.getTracks().forEach((track) => {
         track.stop();
+        didStopMedia = true;
```

Comment:
Not exactly, the issue is that **"media:stopped"** event is triggered regardless if media was stopped or not. By default `didStopMedia` is **false** meaning nothing was stopped, then set to **true** if any track in the stream was stopped.

Reply:
Not sure I agree that's the right approach. We should emit the `media:stopped` event only if we successfully stopped the stream, otherwise we are falsely telling the developer that the streamed has stopped when in case of error it's only partially stopped.

Reply:
Due to the nature of the code, most of the time these streams only contain one track. In fact, it's remoteStream that is the only stream that might contain both audio and video tracks. So we would never be falsely telling the developer. 

The odds of invoking `.getTracks()` having an array of two tracks and looping through and only stopping one of them is very very low.  A `MediaTrack` and `.stop()` method on a track is all native code.

It's really going to be an all or nothing type of deal in reality.